community/imath/build

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+cmake -B build \
+    -DCMAKE_INSTALL_PREFIX=/usr \
+    -DCMAKE_INSTALL_LIBDIR=/lib \
```

Comment:
`LIBDIR` should always be `/usr/lib` -- if the default is `{PREFIX}/lib` in the build system, don't specify this; otherwise `LIBDIR` should be `/usr/lib` or (probably) `lib`.

Reply:
Already fixed to `lib`